### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by sakulstar**
**Things changed:**
1. Small change to game score to make it more readable
2. Installed Svelte French Toast
3. Added Success and Error notifications to app.svelte

### CODE REVIEW

File: src/lib/game.svelte [hunk 1]
```diff
@@ -68,17 +68,14 @@
...
-		</span> 
-	</div>
+<div class="absolute left-0 top-0 font-bold ml-2">
+		<br>
+		Score: {$gameState.score}
+		<br>
+		Number of moves: {$gameState.numberOfMoves}
+		<br>
+		Combo: {$gameState.comboMultiplier}x
+</div>
```
Comment (quabey) [inline src/lib/game.svelte:78]: This is a WIP, if you want to change this. Please build something that is an actual display of these stats. Don't just change the "debug" stats. 

This is something that could be integrated with the timer if you want to work on that too 
Untitled


Can you tell that graphic design is my passion ?

File: src/App.svelte [hunk 2]
```diff
@@ -5,20 +5,24 @@
 	import { onMount } from "svelte";
 	import { authState, loggedIn } from "./lib/store/authState";
 	import { onLogin, onLoggout } from "./lib/supabase.js";
+	import toast, { Toaster } from 'svelte-french-toast';
 
 	onMount(() => {
 		supabase.auth.onAuthStateChange((event, session) => {
 			if (event === 'SIGNED_IN' && !$loggedIn) {
 				$loggedIn = true;
+				toast.success("Logged in");
```
Comment (quabey) [inline src/App.svelte]: Move this to the onLogin function

File: src/App.svelte [hunk 3]
```diff
@@ -5,20 +5,24 @@
...
 				console.log('logged in');
 				try {
 					onLogin();
 				} catch (error) {
+					toast.error("Something went wrong!")
```
Comment (quabey) [inline src/App.svelte:18]: Add this to the onLogin function

File: src/App.svelte [hunk 4]
```diff
@@ -5,20 +5,24 @@
...
 					console.log(error);
 				}
 			}
 			if (event === 'SIGNED_OUT') {
 				$loggedIn = false;
+				toast.success("Logged out");
```
Comment (quabey) [inline src/App.svelte]: Add this to the onLoggout function

**Review by quabey — CHANGES_REQUESTED** [review]

File: src/App.svelte (hunk 3 above)
Comment (sakulstar) [reply src/App.svelte:18]: How would I go on about adding an error to the onLogin function?

Comment (quabey) [reply src/App.svelte:18]: Great question, just leave it there for now, I guess 👍

File: src/lib/game.svelte (hunk 1 above)
Comment (sakulstar) [reply src/lib/game.svelte:78]: Alright, I'll keep that in mind, and I'll definently take a look on integrating some better UI when I get a bit more time 😄

**Review by quabey — APPROVED** [review]

**Pull request merged**